modules/virt-enable-inherit-cpu-model-vms.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-enable-inherit-cpu-model-vms.adoc
+
+[id="virt-enable-inherit-cpu-model-vms_{context}"]
+= Enabling virtual machines to inherit the CPU model
+
+You can schedule a virtual machine (VM) by ensuring that the CPU model for the VM is exactly the same as the CPU model specified for the node. This is done by ensuring that the VM inherits the CPU model specified for the node.
+
+.Prerequisites
+* You configured CPU node discovery.
+
+.Procedure
+
+. Edit the `domain` spec of your VM configuration file. The following example shows `host-passthrough` being specified for the Virtual Machine Instance (VMI):
```

Comment:
`host-passthrough` is a bad example because we don't recommend it.

Please use i.e. `conroe`

Reply:
Removed this content because we don't recommend host-passthrough.